src/interfaces/IData.ts

```diff
@@ -2,4 +2,6 @@ export interface IData {
   categories?: number
   products?: number
   publishedCategories?: number
+  email?: string
+  password?: string
```

Comment:
it is not clear what describe this interface? What is data?

Reply:
This is what the server sends or we send it to the server for verification